go/nbs/store.go

```diff
@@ -381,19 +380,12 @@ func (nbs *NomsBlockStore) Commit(current, last hash.Hash) bool {
 		return true
 	}
 
-	b := &backoff.Backoff{
```

Comment:
Well, i think #3535 is a good place to address that. It was also kind of unfortunate that we'd backoff in the case of getting a manifest-cache oplock failure.